lib/Teamcity/Executor.pm

```diff
@@ -198,6 +203,14 @@ sub touch ($self, $build_name, $properties = {}) {
     return $f;
 }
 
+sub touch_and_no_wait_for_response ($self, $build_name, $properties = {}) {
```

Comment:
I would rather name this method touch_without_future. And the return type of this method is not same as for touch method.